src/mender-update/daemon/context.cpp

```diff
@@ -351,6 +351,68 @@ error::Error Context::SaveDeploymentStateData(StateData &state_data) {
 		return SaveDeploymentStateData(txn, state_data);
 	});
 }
+static error::Error UnmarshalJsonStateDataVersion1(const json::Json &json, StateData &state_data) {
+#define SetOrReturnIfError(dst, expr) \
+	if (!expr) {                      \
+		return expr.error();          \
+	}                                 \
+	dst = expr.value()
```

Comment:
The indentation for `\` is off

Reply:
I see, it is because GH takes tabs as 8 spaces and whoever wrote the original macros (and me) have our editor to 4 spaces. So nothing to do here, it looks good on my machine :sweat_smile:

Reply:
Yeh, I actually tried to align it too, but `clang-format` messes with it anew :laughing:

Reply:
Which is, as you said the nr-spaces ofc :see_no_evil:  Lol, that took me a while! :laughing: